### ISSUE TITLE
Word wrap on search results with long titles

### ISSUE THREAD

**Issue by allanaaa** [omeka-s-themes/foundation#52]
Long titles:

![foundation-titles](https://user-images.githubusercontent.com/28012148/217882356-fdf17812-b4dc-4a4e-8f02-41e94a54753b.PNG)

**Comment by kimisgold** [comment]
I matched these styles to other instances of resource links, and it's ready for a look.

**Comment by allanaaa** [comment]
Ha, I can't seem to generate this precise layout anymore, which might be what you changed and might be some setting I messed up. There is another one though:

![foundation](https://user-images.githubusercontent.com/28012148/219730041-9c6abf3f-bb2d-452c-8696-97d3c4c286e4.PNG)
which is just browsing items in list view:
http://dev.omeka.org/amayer/amayer-s/omeka-s/s/my-omeka-s-site/item

Another thing that's coming up, which might be core and not theme? is the keyword-search box is coming up twice when you do a simple search and then click through to the results in item sets:
http://dev.omeka.org/amayer/amayer-s/omeka-s/s/my-omeka-s-site/item-set?fulltext_search=scottish
![foundation2](https://user-images.githubusercontent.com/28012148/219731474-b35ab81d-f71e-48e8-bb46-2d7f542047f8.PNG)

I did check it in one other theme and it wasn't there, but I'm not clear on what's happening in the backend to make it appear.
